Lib/socketserver.py

```diff
@@ -221,10 +222,6 @@ def serve_forever(self, poll_interval=0.5):
         """
         self.__is_shut_down.clear()
         try:
-            # XXX: Consider using another file descriptor or connecting to the
```

Comment:
In my opinion, don't need to remove this comment because the proposal has not yet been resolved.

Reply:
I added the comment again, but I do think that by implementing the server_shutdown() function this is resolved